src/stores/context.store.ts

```diff
@@ -9,6 +9,7 @@ type Classification = {
 export interface IDataset {
     datasetId: string;
     apiKey: string;
+    serverUrl: string;
```

Comment:
```suggestion
    serverUrl?: string;
```
Lad serverurl være optionel